pkgs/tools/audio/liquidsoap/full.nix

```diff
@@ -1,8 +1,8 @@
 { lib, stdenv, makeWrapper, fetchurl, which, pkg-config
 , libjpeg
 , ocamlPackages
-, awscli2, curl, ffmpeg, youtube-dl
-, runtimePackages ? [ awscli2 curl ffmpeg youtube-dl ]
+, awscli2, curl, ffmpeg
+, runtimePackages ? [ awscli2 curl ffmpeg ]
```

Comment:
```suggestion
, awscli2, curl, ffmpeg
, withAws ? true
, withCurl ? true
, withFfmpeg ? true
, withYoutubeDl ? true
, extraRuntimePackages ? [  ]
```

Reply:
By default liquidsoaps youtube protocol only looks for the `youtube-dl` binary in liquidsoap scripts one can use `settings.protocol.youtube_dl.path.set("yt-dlp")` to switch over.

The list included in the runtime packages are what liquid-soap supports out of the box with no settings set.

Reply:
That's what I mean; can it find and use yt-dlp OOTB?

Reply:
I use it like this here: `pkgs.writers.writeDashBin "youtube-dl" ''${pkgs.yt-dlp}/bin/yt-dlp "$@"''`. but this is not something we should do here IMHO